assets/pages/oneoff-contributions/helpers/ajax.js

```diff
@@ -41,20 +44,27 @@ type OneoffContribFields = {
 function requestData(paymentToken: string, getState: () => PageState) {
 
   const state = getState();
+  const user = state.page.user;
+  const ophanIds: OphanIds = getOphanIds();
+
+  if (user.fullName !== null && user.fullName !== undefined &&
+    user.email !== null && user.email !== undefined) {
+    const referrerAcquisitionData = state.common.referrerAcquisitionData;
 
-  if (state.page.user.fullName !== null && state.page.user.fullName !== undefined
-    && state.page.user.email !== null && state.page.user.email !== undefined) {
     const oneOffContribFields: OneoffContribFields = {
```

Comment:
@Mullefa @desbo We are sending this data to contributions' one-off backend. Is something needed?

Reply:
This is ok, but we include referrer acquisition data in an addition request? See [`ContributionRequest`](https://github.com/guardian/contributions-frontend/blob/master/app/controllers/forms/ContributionRequest.scala) for all fields currently accepted.